pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -174,6 +175,8 @@ cdef convert_to_timedelta64(object ts, object unit):
 
     Return an ns based int64
     """
+    if unit is None:
+        unit = 'ns'
```

Comment:
where is this called from?  could we type the argument as `str` and require the caller to pass it?

Reply:
e.g. in array_to_timedelta64 below we should just do that check once at the top of the function, not in every step